pkg/volume/iscsi/iscsi_util.go

```diff
@@ -50,7 +50,7 @@ const (
 	maxAttachAttempts = 5
 
 	// How many seconds to wait for a multipath device if at least two paths are available.
-	multipathDeviceTimeout = 10
+	multipathDeviceTimeout = 30
```

Comment:
I'm re-reading the code again and it's a bit confusing. Constant name and the comment above do not correspond to how it is used in the code.

As a fix I would propose:

* Use `multipathDeviceTimeout` instead of hardcoded `10` here: https://github.com/kubernetes/kubernetes/blob/bdd5c15dee916dbea9d50dd678cdc7c8e1c49901/pkg/volume/iscsi/iscsi_util.go#L460 - this is the place where kubelet waits for multipath device to appear.

* Create a new constant for waiting for a single path to appear and use it instead of `multipathDeviceTimeout` everywhere else. IMO 30s is fine.

Reply:
@jsafrane I see, indeed its bit confusing at current state. I have accomodated your review comment and updated the patchset. Hopefully thats what you meant. Please let me know if I missed any or misinterpreted anything.